metadata-io/src/test/java/com/linkedin/metadata/ESSampleDataFixture.java

```diff
@@ -182,7 +184,16 @@ protected EntityClient entityClient(
                 false);
 
         AspectDao mockAspectDao = mock(AspectDao.class);
-        when(mockAspectDao.batchGet(anySet())).thenReturn(Map.of(mock(EntityAspectIdentifier.class), mock(EntityAspect.class)));
+        when(mockAspectDao.batchGet(anySet())).thenAnswer(i -> {
```

Comment:
Are there cases besides the "exists" case where this batchGet will be called within the scope of the test that use this fixture? 

This seems like a broad solution to the fact that we are now using this API

Reply:
Yes, this is a more general solution indeed because it reuses the parameter of the call instead of hardcoding some predefined values.

Besides the new `exists` method, I don't think any other tests using this fixture care about the response of the `batchGet`.